test/stream/stats.js

```diff
@@ -5,92 +5,47 @@ var stats = require('../../stream/stats'),
 module.exports.tests = {};
```

Comment:
I don't see stats being used in the code anymore (other than tests). Can it be removed altogether?